aws/templates/common.ftl

```diff
@@ -1499,12 +1495,29 @@ behaviour.
                                     "Id" : formatId(subComponentId),
                                     "Name" : formatName(subComponentName)
                                 }
-                            ),
+                            )
+                        }
+                    ]
+
+                    [#local occurrenceProfiles = [] ]
+                    [#list occurrenceContexts as occurrenceContext ]
+                        [#if ((occurrenceContext.Profiles.Deployment)![])?has_content ]
+                            [#local occurrenceProfiles = occurrenceContext.Profiles.Deployment ]
+                        [/#if]
+                    [/#list]
+
+                    [#if occurrenceProfiles?has_content ]
```

Comment:
I think you don't want this test as it will then not pick up profiles set at other levels like product or tenant.

I think you could just skip the if test and pass in the potentially empty array.

Effectively we want deployment profiles to trump everything.